test/message_bus/Proof/setup_proof.js

```diff
@@ -0,0 +1,114 @@
+// Copyright 2020 OpenST Ltd.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+'use strict';
+
+const Proof = artifacts.require('ProofDouble');
+const BN = require('bn.js');
+
+const { AccountProvider } = require('../../test_lib/utils.js');
+const Web3 = require('../../test_lib/web3.js');
+const Utils = require('../../test_lib/utils.js');
+
+contract('Proof::setupProof', (accounts) => {
+  const accountProvider = new AccountProvider(accounts);
+  let proof;
+  let setupParams;
+
+  beforeEach(async () => {
+    proof = await Proof.new();
+
+    setupParams = {
+      storageAccount: accountProvider.get(),
+      stateRootProvider: accountProvider.get(),
+      maxStorageRootItems: new BN(100),
+    };
+  });
+
+
+  contract('Negative Tests', async () => {
+    it('should fail when stateRootProvider address is empty', async () => {
```

Comment:
I think statement and test is not aligned.

Reply:
Corrected.